CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ You can find its changes [documented below](#031-2024-07-29).
 
 This release has an [MSRV][] of 1.75.
 
+## [0.5.0][] (2024-11-16)
```

Comment:
You need to update the paragraph above "Unreleased" to indicate the new release and link to the changes.

You also need to add in the link definition for the PR #42 as you missed it when doing the changelog entry.

Reply:
You'll also need various entries down at the bottom for the changes.

Reply:
Not convinced all this cross-linking is worth the effort, but for now I've updated. I believe that issue and github username links could be made simpler by simply typing things like #42 and @nicoburns as plain text. Github will autolink these.

Reply:
Oh? Is that a setting? I've been annoyed by changelogs which don't have this linking before. For example, https://github.com/gfx-rs/wgpu/blob/c110bf22d811ac1a65d79375e4cfa2313787e3bc/CHANGELOG.md#new-features doesn't link to the users (I've also seen it with numbers, but don't have an example to hand).

From my experience, the autolinks seem to only apply to comments/other types of "conversation", not rendered markdown files. But if there's a way to enable it markdown files, I'd be interested in seeing it. If it's a per-user setting, we should probably document it at the top of the changelog.

Reply:
What a utopia we would be living in if autolinking applied to Changelogs...

Reply:
> From my experience, the autolinks seem to only apply to comments/other types of "conversation", not rendered markdown files.

Ah, I think you're probably right. Maybe something we could request. But I guess it could get annoying for non-github related markdown files.